### PR TITLE
Unify prop names + Extract and refector Portal component

### PR DESCRIPTION
**Pull request by stuniel**

### CODE REVIEW

**Comment by stuniel** [comment]
Closes #41

File: packages/modal/src/Modal.js [hunk 1]
```diff
@@ -1,90 +1,53 @@
 import React from 'react'
-import ReactDOM from 'react-dom'
 import PropTypes from 'prop-types'
 import cls from 'classnames'
+import Portal from './Portal'
```
Comment (rangoo94) [inline packages/modal/src/Modal.js]: AFAIR Portal had to be extracted to separated package, am I right @michalochman ?

File: packages/modal/src/Modal.js [hunk 2]
```diff
@@ -108,14 +71,14 @@ Modal.propTypes = {
   children: PropTypes.node,
 
   /** Controls whether modal is open */
-  isOpen: PropTypes.bool,
+  open: PropTypes.bool,
 
   /** Root element of modal */
-  root: PropTypes.object
+  attachTo: PropTypes.object
```
Comment (rangoo94) [inline packages/modal/src/Modal.js]: Good name :+1:

File: packages/modal/src/Portal.js [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+ * @property {*} [props.children]
+ * @class
+ */
+class Portal extends React.Component {
+  constructor (props) {
+    super(props)
+    this.el = document.createElement('div')
+  }
+
+  componentDidMount () {
```
Comment (rangoo94) [inline packages/modal/src/Portal.js]: If we have proper `Portal` component maybe it's worth to handle a change of `attachTo` component?

File: packages/modal/src/Modal.js (hunk 1 above)
Comment (michalochman) [reply packages/modal/src/Modal.js]: We decided to extract it with next PR.

File: packages/modal/src/Portal.js (hunk 3 above)
Comment (rangoo94) [reply packages/modal/src/Portal.js]: Also, shouldn't we extract is as separated package as it's used in many others?

Comment (michalochman) [reply packages/modal/src/Portal.js]: Let's extract it after it's merged.

**Comment by rangoo94** [comment]
@stuniel, could you resolve conflicts to merge it? ;)

File: packages/modal/src/Portal.js (hunk 3 above)
Comment (rangoo94) [reply packages/modal/src/Portal.js]: As a Portal is here exposed in `Popover`, I will do it ;)

**Pull request merged**